azure-monitor-opentelemetry-distro/README.md

```diff
@@ -28,6 +28,18 @@ Install the Azure Monitor Opentelemetry Distro with [pip][pip]:
 pip install azure-monitor-opentelemetry-distro --pre
 ```
 
+### Manual Configuration
+
+You can use the `configure_opentelemetry` to set up manual instrumentation in a single line and send telemetry for your app to Azure Monitor. `configure_opentelemetry` supports the following optional arguments:
+
```

Comment:
No need to mention specifically about exporter implementation detail. Just say: `connection string will be automatically populated from the APPLICATIONINSIGHTS_CONNECTION_STRING environment variable if not explicitly passed in.`

Reply:
Done